packages/postcss-svgo/package.json

```diff
@@ -28,16 +28,15 @@
   "repository": "cssnano/cssnano",
   "dependencies": {
     "postcss-value-parser": "^4.2.0",
-    "svgo": "^2.7.0"
+    "svgo": "^3.0.0"
   },
   "bugs": {
     "url": "https://github.com/cssnano/cssnano/issues"
   },
   "engines": {
-    "node": "^10 || ^12 || >=14.0"
+    "node": ">=14.0"
```

Comment:
https://github.com/svg/svgo/releases/tag/v3.0.0 lists this as a breaking change.